### ISSUE TITLE
No-code quick task: help renaming `master` to `main`?

### ISSUE THREAD

**Issue by rayrrr** [jazzband-roadies/help#325]
In the Jazzband-managed project https://github.com/jazzband/geojson we have an [open issue](https://github.com/jazzband/geojson/issues/181) to rename our default branch from `master` to `main`.

Although I am the project lead, I believe only the roadies have the access needed to make "main" the new default branch within GitHub as per [these instructions from GitHub](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-branches-in-your-repository/renaming-a-branch).

TLDR; A roadie should be able to view the [branch list](https://github.com/jazzband/geojson/branches), edit the branch called `master` by clicking the pencil icon inline in the tabular display for that branch, and rename it `main` in an elegant manner with a few clicks and keystrokes. Please assist. TIA

**Comment by jezdez** [comment]
This is done!

**Comment by rayrrr** [comment]
Thank you @jezdez
